pygame 2.0.0 (SDL 2.0.12, python 3.7.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
boxing_v1 pettingzoo
random seed: [60, 94, 32, 49]
<SubprocVectorEnv instance>
Traceback (most recent call last):
  File "general_run.py", line 43, in <module>
    launch_rollout(parser_args.env, parser_args.method)
  File "general_run.py", line 22, in launch_rollout
    model1 = eval(args.algorithm)(env, args)
  File "/home/zihan/research/MARS/mars_core/rl/algorithm/nash_dqn_exploiter.py", line 54, in __init__
    self.debugger = Debugger(env, "./data/nash_dqn_exploiter_simple_mdp_log.pkl")
  File "/home/zihan/research/MARS/mars_core/rl/algorithm/nash_dqn.py", line 35, in __init__
    self.num_states_per_step = int(self.env.observation_space.shape[0])
AttributeError: 'list' object has no attribute 'shape'
